contribute/ISSUE_TRIAGE.md

```diff
@@ -188,54 +189,54 @@ If it's not perfectly clear that it's an actual bug, quickly try to reproduce it
 1. Either [ask for more information](#2-ensure-the-issue-contains-basic-information) needed to investigate it more thoroughly.
 2. Either [delegate further investigations](#investigation-of-issues) to someone else.
 
-**It works as intended/by design:**
+**It works as intended (that is, by design):**
 
 1. Kindly and politely add a comment explaining briefly why we think it works as intended and close the issue.
 2. Label the issue `type/works-as-intended`.
 
-### Enhancement/feature?
+#### Enhancement or feature request?
 
 1. Label the issue `type/feature-request` and add at least one `area/*` or `datasource/*` label.
 2. Make sure the submitter has justified why this feature requests is important.
 
-### Documentation issue?
+#### Documentation issue?
 
 First, evaluate if the documentation makes sense to be included in the Grafana project:
 
-- Is this something we want/can maintain as a project?
-- Is this referring to usage of some specific integration/tool and in that case is that a popular use case in combination with Grafana?
-- If unsure, kindly and politely add a comment explaining that we would need [upvotes](https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments) to identify that lots of other users want/need this.
+- Is this something we want and can maintain as a project?
+- Is this referring to usage of some specific integration and in that case is that a popular use case in combination with Grafana?
+- If unsure, kindly and politely add a comment explaining that we would need [upvotes](https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments) to identify that lots of other users want or need this.
 
 Second, label the issue `type/docs` and at least one `area/*` or `datasource/*` label.
 
 **Minor typo/error/lack of information:**
 
-There's a minor typo/error/lack of information that adds a lot of confusion for users and given the amount of work is a big win to make sure fixing it:
+There's a minor typo, error, or lack of information that adds a lot of confusion for users and is a big win to fix it:
 
 1. Either update the documentation yourself and open a pull request.
-2. Either delegate the work to someone else by assigning that person to the issue and add the issue to next major/minor milestone.
+2. Either delegate the work to someone else by assigning that person to the issue. Note that the milestone is automatically assigned and usually doesn't need to be manually edited.
```

Comment:
should use lazy numbering here

Reply:
good catch. i have added lazy numbers throughout doc